draft-ietf-tls-dtls13.md

```diff
@@ -1130,12 +1132,14 @@ fragmentation, DTLS modifies the TLS 1.3 handshake header:
             case client_hello:          ClientHello;
             case server_hello:          ServerHello;
             case end_of_early_data:     EndOfEarlyData;
+            case new_session_ticket:    NewSessionTicket;
             case encrypted_extensions:  EncryptedExtensions;
-            case certificate_request:   CertificateRequest;
+            case request_connection_id: RequestedConnectionId;
```

Comment:
Shouldn't this be just "RequestConnectionId", not "Requested"?

Reply:
Yes fixing this too.